src/erlfmt.erl

```diff
@@ -188,6 +189,32 @@ remove_pragma_comment_block([Head | Tail]) ->
         _ -> Tail
     end.
 
+replace_pragma_node(Node0) ->
+    {PreComments0, _, PostComments0} = erlfmt_format:comments_with_pre_dot(Node0),
+    PreComments = replace_pragma_comment_blocks("%%%", PreComments0),
+    PostComments = replace_pragma_comment_blocks("%%%", PostComments0),
+    Node = erlfmt_scan:put_anno(pre_comments, PreComments, Node0),
+    erlfmt_scan:put_anno(post_comments, PostComments, Node).
+
+replace_pragma_comment_blocks(_Prefix, []) ->
+    [];
+replace_pragma_comment_blocks(Prefix, [{comment, Loc, Comments} | Rest]) ->
+    case replace_pragma_comment_block(Prefix, Comments) of
+        [] ->
+            replace_pragma_comment_block(Prefix, Rest);
```

Comment:
Can this happen? I don't think we ever produce empty comment blocks and `replace_pragma_comment_block/2` never drops any list elements

Reply:
Good catch, I updated the code.